include/odp/api/spec/crypto.h

```diff
@@ -585,15 +585,34 @@ int odp_crypto_auth_capability(odp_auth_alg_t auth,
  * default values.
  *
  * @param param             Session parameters
- * @param session           Created session else ODP_CRYPTO_SESSION_INVALID
  * @param status            Failure code if unsuccessful
  *
- * @retval 0 on success
- * @retval <0 on failure
+ * @retval created session on success
+ * @retval ODP_CRYPTO_SESSION_INVALID on failure
  */
+odp_crypto_session_t odp_crypto_session_create2(odp_crypto_session_param_t
```

Comment:
I agree that this is a poor choice of name that may offer some short term benefits at long term cost. If we need to make this change then it's probably better to just keep the same name and not worry about deprecation.